tests/providers/test_python.py

```diff
@@ -3,13 +3,46 @@
 import unittest
 import warnings
 
+from typing import Any, Dict, Iterable, Optional, Union
 from unittest.mock import patch
 
 import pytest
 
 from faker import Faker
 
 
+@pytest.mark.parametrize(
+    "object_type, kwargs",
+    (
+        (None, {}),
+        (bool, {}),
+        (str, {}),
+        (float, {}),
+        (int, {}),
+        (tuple, {}),
+        (set, {}),
+        (list, {}),
+        (Iterable, {}),
+        (dict, {}),
+    ),
+)
```

Comment:
I dont see the point in defining `kwargs`, since it's always an empty dict. We could just have:

```suggestion
@pytest.mark.parametrize(
    "object_type",
    (
        (None,),
        (bool,),
        (str,),
        (float,),
        (int,),
        (tuple,),
        (set,),
        (list,),
        (Iterable,),
        (dict,),
    ),
)
```